components/homme/src/theta-l/prim_advance_mod.F90

```diff
@@ -464,6 +509,7 @@ subroutine applyCAMforcing_dynamics(elem,hvcoord,np1,dt,nets,nete)
   do ie=nets,nete
 
      elem(ie)%state%vtheta_dp(:,:,:,np1) = elem(ie)%state%vtheta_dp(:,:,:,np1) + dt*elem(ie)%derived%FT(:,:,:)
+     elem(ie)%state%phinh_i(:,:,1:nlev,np1) = elem(ie)%state%phinh_i(:,:,1:nlev,np1) + dt*elem(ie)%derived%FPHI(:,:,1:nlev)
```

Comment:
i assume you ran dcmip16 tests with forcings on phi. should i could run them for ftype4?